jax/api.py

```diff
@@ -1120,7 +1120,7 @@ def _shape_spec_consistent(spec, expr):
   return all(a == b for a, b in zip(spec, expr) if a is not masking.monomorphic_dim)
 
 
-def jvp(fun: Callable, primals, tangents):
+def jvp(fun: Union[Callable, lu.WrappedFun], primals, tangents):
```

Comment:
I feel that adding `lu.WrappedFun` here pollutes the API somewhat. I presume that this is needed because we call this API function internally on a WrappedFun. Is there a way to avoid that?

Reply:
I split the function into `_jvp` and `jvp`, the former being an internal function that takes a `WrappedFun`.